### DISCUSSION TITLE
Install problem

### DISCUSSION THREAD

**Discussion by rafassotero** [MuiseDestiny/zotero-gpt#125] [Q&A] [answered]
Hey there, i can't install the zip file on Zotero. 

I went for "install addon from file", choose the zip file i downloaded here but it says "the extra "%S" can't be installed. It could be incompatible with this version of Zotero"

Any thoughts? Thanks

**Comment by MuiseDestiny** [comment, marked as answer]
XPI file. https://github.com/MuiseDestiny/zotero-gpt/releases/download/0.2.5/zotero-gpt.xpi

**Comment by rafassotero** [comment]
thanks!
